### PR TITLE
feat: add status support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/exportManager/interfaces.ts [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
 import { CreateExportTaskExtendedRequest } from '../tasks/models/tasksManager';
+import { ArtifactType } from './enum';
+import { OperationStatus } from '../tasks/enums';
 
 export interface Webhook {
   events: TaskEvent[];
   url: string;
 }
+
+export interface IArtifactDefinition {
```
Comment (asafMasa) [inline src/exportManager/interfaces.ts:14]: isn't it in @mapcolonies/types package?

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: src/clients/jobManager/enums.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export enum OperationStatus {
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply src/clients/jobManager/enums.ts:1]: fixed

File: src/tasks/models/tasksManager.ts [hunk 3]
```diff
@@ -63,7 +64,7 @@ export class TasksManager {
       artifactCRS: EPSGDATA[4326].code,
       description: params.description,
       keywords: jobParameters.keywords,
-      status: params.status,
+      status: convertToUnifiedTaskStatus(params.status),
```
Comment (asafMasa) [inline src/tasks/models/tasksManager.ts:67]: Check isCleaned in job to return appropriate status, ask @syncush if we need to remove artifact URI as well

Comment (asafMasa) [inline src/tasks/models/tasksManager.ts:67]: Check isCleaned in job to return appropriate status

Comment (asafMasa) [reply src/tasks/models/tasksManager.ts:67]: E2E check that response cahanges for cleaned artifact

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: src/tasks/models/tasksManager.ts (hunk 3 above)
Comment (asafMasa) [reply src/tasks/models/tasksManager.ts:67]: Check for precentage change while task is running

**Review by asafMasa — APPROVED** [review]
😲

**Pull request merged**